File: app/src/main/java/com/fuchsia/shotokanwkf/MainAd.java
package com.fuchsia.shotokanwkf;

import android.os.Bundle;
import android.os.PersistableBundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class MainAd extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Admob.loadInter(MainAd.this);

    }
}

File: app/src/main/java/com/fuchsia/shotokanwkf/MainAd.kt
package com.fuchsia.shotokanwkf

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.fuchsia.shotokanwkf.Admob.loadInter

open class MainAd : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        loadInter(this@MainAd)
    }
}